ui/admin/tests/e2e/tests/worker-ent.spec.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/* eslint-disable no-undef */
+const { expect } = require('@playwright/test');
+const { test } = require('@playwright/test');
+const { authenticatedState } = require('../helpers/general');
+
+test.use({ storageState: authenticatedState });
+
+test('Create a worker (enterprise) @ent @docker @aws', async ({ page }) => {
+  await page.goto('/');
+  await page
+    .getByRole('navigation', { name: 'General' })
+    .getByRole('link', { name: 'Workers' })
+    .click();
+  await page.getByRole('link', { name: 'New', exact: true }).click();
+
+  // Populate config fields
+  await page.getByLabel('Worker public address').fill('worker1.example.com');
+  await page.getByLabel('Config file path').fill('/home/ubuntu/boundary');
+  await page.getByLabel('Initial upstreams').fill('10.0.0.1, 10.0.0.2');
+  await page
+    .getByRole('group', { name: 'Worker Tags' })
+    .getByLabel('Key')
+    .fill('type');
+  await page
+    .getByRole('group', { name: 'Worker Tags' })
+    .getByLabel('Value')
+    .fill('downstream, pki');
+  await page
+    .getByRole('group', { name: 'Worker Tags' })
+    .getByRole('button', { name: 'Add' })
+    .click();
+  await page
+    .getByRole('group', { name: 'Worker Tags' })
+    .getByLabel('Key')
+    .last()
+    .fill('test');
+  await page
+    .getByRole('group', { name: 'Worker Tags' })
+    .getByLabel('Value')
+    .last()
+    .fill('example');
+  await page
+    .getByRole('group', { name: 'Worker Tags' })
+    .getByRole('button', { name: 'Add' })
+    .click();
+
+  await page.getByRole('switch', { name: 'Enable Recording Storage' }).click();
+  await page.getByLabel('Recording Storage Path').fill('/tmp/recordings');
+
+  // Check auto-populated config
+  const configLocator = '[data-test-worker-config=""]';
```

Comment:
When selecting `data-test-*` selectors, since we do not follow a key/value approach (i.e: data-test-worker-config='value') I will recomend to use just the key as selector, i.e: `[data-test-worker-config]`. [Other examples](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/storage-buckets/selectors.js#L35)